src/utils/mux-data-middleware.js

```diff
@@ -36,7 +36,8 @@ function setupMuxDataMetadataOverride (videoEl, options) {
 }
 
 function injectMuxDataMetadata (options) {
-  options.plugins.mux.data["player_software_name"] = "vjs-mux-kit-" + videojs?.VERSION.split('.')[0];
+  // The default build includes Hls and it's a function, in the VHS build Hls.js is excluded and it's an empty object
+  options.plugins.mux.data["player_software_name"] = "vjs-mux-kit-" + videojs?.VERSION.split('.')[0] + (typeof Hls === 'function' ? '' : '-vhs');
```

Comment:
We may want to be explicit here:

`vjs-mux-kit-7-hlsjs` vs. `vjs-mux-kit-7-vhs`

Reply:
![vhs_2022-02-03_16-23-41](https://user-images.githubusercontent.com/764988/152451496-2f6ab75b-dc46-4613-a05f-885a6f8fda4a.png)

👍🏼

Reply:
My thought was keeping the hls.js build without any changes as much as possible, particularly if someone is already using it so that the name won't change in Mux data. But I can add `hlsjs` if you'd like.

Reply:
That is valid. Given that this lib is still relatively young and it's a minor detail I think I favor being explicit about which build is being used. The only real reason is to avoid ambiguity when looking at the data.

If we see `vjs-mux-kit-7` in the data for "Software Name" the next question is "Is that the HLS build or the VHS build?" -- the answer to that question would be non-obvious. 

For that reason I like:

- `vjs-mux-kit-7-hlsjs` and `vjs-mux-kit-7-vhs`

Reply:
Sounds good to me.

Reply:
Updated!